### ISSUE TITLE
nzbToMedia directory empty

### ISSUE THREAD

**Issue by andyj682** [SiCKRAGETV/sickrage-issues#3100]
I was going to play around with nzbToMedia today to try to get it working before autoProcessTV goes away, but the "nzbTomedia" directory in contrib is empty. Confirmed that I've got the latest version of SR, etc. etc. Running on a Synology NAS. Any ideas? I know I can just download it separately, but I'm assuming it's actually supposed to be in contrib, right?

**Comment by miigotu** [comment]
hmm, the update is supposed to also run `git submodule init --update`

**Comment by andyj682** [comment]
@miigotu  tried git submodule update --init and got this:

/usr/local/git/libexec/git-core/git-submodule: line 1: /usr/bin/perl: not found

Turns out perl is not part of Synology DSM. There is an add-on package, though--installed that, ran submodule update, and now everything's great. But I assume most people running on Synology are going to be in for the same surprise I was.

TL;DR: If you're running SR on Synology, install perl.

**Comment by andyj682** [comment]
(Also weird that git submodule depends on perl when the rest of it doesn't, but whatever.)

**Comment by miigotu** [comment]
Very weird. Will see if there is another option.
